### PR TITLE
holesky: switch to nftables

### PR DESCRIPTION
**Pull request by mendelskiv93**
Switch `nimbus.holesky` hosts to nftables.

- [x] `erigon`
- [x] `nethermind` 
- [x] `geth` 

Reference: https://github.com/status-im/infra-misc/issues/301

### CODE REVIEW

File: ansible/host_vars/geth-01.ih-eu-mda1.nimbus.holesky.yml [hunk 1]
```diff
@@ -13,9 +13,10 @@ test_api_beacon_node_branch: |-
 
 # WARNING: This will change if number of nodes changes.
 redirect_ports:
-  - src: 80
+  public-beacon-api:
+    src: 80
     dst: '{{ test_api_beacon_node_rest_port }}'
-    comment: 'Test Beacon API ({{ test_api_beacon_node_branch }})'
+    comment: 'Test Beacon API ({{ test_api_beacon_node_branch }}-0{{ test_api_beacon_node_rest_index|int - 1 }})'
```
Comment (jakubgs) [inline ansible/host_vars/geth-01.ih-eu-mda1.nimbus.holesky.yml:19]: You don't need this here, there's no indexes in node names in this fleet.

File: ansible/host_vars/geth-02.ih-eu-mda1.nimbus.holesky.yml [hunk 2]
```diff
@@ -13,6 +13,7 @@ test_api_beacon_node_branch: |-
 
 # WARNING: This will change if number of nodes changes.
 redirect_ports:
-  - src: 80
+  public-beacon-api:
+    src: 80
     dst: '{{ test_api_beacon_node_rest_port }}'
-    comment: 'Test Beacon API ({{ test_api_beacon_node_branch }})'
+    comment: 'Test Beacon API ({{ test_api_beacon_node_branch }}-0{{ test_api_beacon_node_rest_index|int - 1 }})'
```
Comment (jakubgs) [inline ansible/host_vars/geth-02.ih-eu-mda1.nimbus.holesky.yml:19]: You don't need this here, there's no indexes in node names in this fleet.

File: ansible/group_vars/nimbus.holesky.yml [hunk 3]
```diff
@@ -1,6 +1,6 @@
 ---
 # Ports used for all 3 EL nodes as they are mutually exclusive.
-exec_layer_p2p_port:     '{{ 30303 + 10*idx|int + 1 }}'
+exec_layer_p2p_port:     '{{ 30303 + 3*idx|int + 1 }}'
```
Comment (jakubgs) [inline ansible/group_vars/nimbus.holesky.yml:3]: Fix indentation.

File: ansible/group_vars/nimbus.holesky.yml [hunk 4]
```diff
@@ -153,13 +153,27 @@ nimbus_era_files_network: '{{ beacon_node_network }}'
...
-  - { port: '6060:6070',   ipset: 'metrics.hq',  comment: 'EL Metrics' }
-  - { port: '9400:9404',   ipset: 'metrics.hq',  comment: 'Geth Exporter' }
-  - { port: '9300:9310',   ipset: '{{ env }}.{{ stage }}', comment: 'Nimbus REST API' }
+  nginx:
+    - { port: 443, comment: 'Nginx' }
+  exec-node:
+    - { port: '30303-30315', comment: '{{ host_el_type }} libp2p stable',    protocol: 'tcp'                     }
+    - { port: '30303-30315', comment: '{{ host_el_type }} discovery stable', protocol: 'udp'                     }
+    - { port: '6060-6064',   comment: 'EL Exporter',                         ipset: 'metrics.hq', iifname: 'wg0' }
+    - { port: '9400-9404',   comment: 'Go-Ethereum Exporter',                ipset: 'metrics.hq', iifname: 'wg0' }
```
Comment (jakubgs) [inline ansible/group_vars/nimbus.holesky.yml:169]: What's the `stable` part for tho?
```suggestion
    - { port: '30303-30315', comment: '{{ host_el_type }} LibP2P',    protocol: 'tcp'                     }
    - { port: '30303-30315', comment: '{{ host_el_type }} Discovery', protocol: 'udp'                     }
    - { port: '6060-6064',   comment: '{{ host_el_type }} Metrics',                         ipset: 'metrics.hq', iifname: 'wg0' }
    - { port: '9400-9404',   comment: '{{ host_el_type }} Exporter',                ipset: 'metrics.hq', iifname: 'wg0' }
```

**Review by jakubgs — CHANGES_REQUESTED** [review]

**Review by yakimant — APPROVED** [review]

File: ansible/group_vars/nimbus.holesky.yml [hunk 5]
```diff
@@ -153,13 +153,27 @@ nimbus_era_files_network: '{{ beacon_node_network }}'
...
-  - { port: '6060:6070',   ipset: 'metrics.hq',  comment: 'EL Metrics' }
-  - { port: '9400:9404',   ipset: 'metrics.hq',  comment: 'Geth Exporter' }
-  - { port: '9300:9310',   ipset: '{{ env }}.{{ stage }}', comment: 'Nimbus REST API' }
+  nginx:
+    - { port: 443, comment: 'Nginx' }
+  exec-node:
+    - { port: '30303-30315', comment: '{{ host_el_type }} LibP2P',    protocol: 'tcp'                     }
+    - { port: '30303-30315', comment: '{{ host_el_type }} Discovery', protocol: 'udp'                     }
+    - { port: '6060-6064',   comment: '{{ host_el_type }} Metrics',   ipset: 'metrics.hq', iifname: 'wg0' }
+    - { port: '9400-9404',   comment: '{{ host_el_type }} Exporter',  ipset: 'metrics.hq', iifname: 'wg0' }
```
Comment (yakimant) [inline ansible/group_vars/nimbus.holesky.yml:169]: 6061-6064 ?
30304, 30307, 30310, 30313 ?

**Review by yakimant — CHANGES_REQUESTED** [review]
can you please check the port brackets?

File: ansible/group_vars/nimbus.holesky.yml (hunk 5 above)
Comment (mendelskiv93) [reply ansible/group_vars/nimbus.holesky.yml:169]: we could tighten a little bit but `erigon` uses two ports for every stage so this was a "fit all" solution and we left additional port open for future development

6061-6064 -> good catch

**Review by jakubgs — APPROVED** [review]

**Pull request merged**